target/linux/lantiq/image/netgear.mk

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2017 LEDE & Edward O'Callaghan
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+DEVICE_VARS += NETGEAR_BOARD_ID NETGEAR_REGION NETGEAR_HW_ID
+
+define Build/append-hdr-rootfs
+  mkimage -A mips -O linux -C lzma -T filesystem -a 0x00 -e 0x00 \
+          -n 'LEDE RootFS' \
+          -d $(IMAGE_ROOTFS) $@.tmp
+  cat $@.tmp >> $@
+  rm $@.tmp
+endef
+
+define Device/lantiqNetgear
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma | pad-offset 4 0 | pad-to 2162624
+  IMAGES := sysupgrade.bin factory.img
+  IMAGE/default := append-kernel | append-hdr-rootfs | pad-rootfs | append-metadata
```

Comment:
I'm not yet sure what append-hdr-rootfs really does, but it look similar to the netgear fake rootfs header. Does using [append-uImage-fakeroot-hdr](https://github.com/lede-project/source/commit/7d00cfe9bb693e376ac9d035e13f8ce8a5ff572c) work as well?